packages/lib/src/bleed/Bleed.tsx

```diff
@@ -1,8 +1,7 @@
 import styled from "styled-components";
 import BleedPropsType from "./types";
-import { CoreSpacingTokensType } from "../common/coreTokens";
 
-const getSpacingValue = (spacingName?: CoreSpacingTokensType) => (spacingName ?? "0rem");
+const getSpacingValue = (spacingName?: string) => spacingName ?? "0rem";
 
 const StyledBleed = styled.div<BleedPropsType>`
```

Comment:
Can we refactor and replace the `StyledBleed` with our `DxcContainer`, as in the Inset?